index.js

```diff
@@ -0,0 +1,31 @@
+const langSec = document.querySelector(".language-section");
+const buttons = document.querySelectorAll(".button");
+const textTitle = document.querySelector(".title");
+const textDesc = document.querySelector(".description");
+
+buttons.forEach(button => {
+    button.addEventListener('click', () => {
+        langSec.querySelector(".active").classList.remove(".active");
+        button.classList.add("active");
+
+        const attr = button.getAttribute("data-language");
+
+        textTitle.textContent = data[attr].title;
+        textDesc.textContent = data[attr].description;
+    });
+});
+
+const data = {
+    "english": {
+        "title": "Translating static text.",
+        "description": "Translating static text"
+    },
+    "french": {
+        "title": "Traduire du texte statique.",
+        "description": "Traduire du texte statique."
+    },
+    "german": {
+        "title": "Statischen Text übersetzen.",
+        "description": "Statischen Text übersetzen."
+    }
```

Comment:
If you wanted to store your dictionary in a JSON file, or a series of JSON files (`en.json`, `de.json`, `fr.json`) then this double-quotes surrounding everything approach would be the right (only) way to do it.

And on a larger project, where all the copy on the site/app amounts to hundreds of KB or even MB's of data, then it would be much more preferred to break it up. Imagine going to a site and downloading a 1MB file for English, and then having to wait for 40 more MB to download of every other language, when you'll never switch to them. So it is more performant to break them up into language specific files, so they can be loaded asynchronously, on big projects.

But that's overkill for this project.

I think it's fine to keep this here in this file, but we should change it to be written more like what most devs would do. Switch to single-quotes, and remove quotes from the keys if they aren't needed.

```suggestion
  english: {
    title: 'Translating static text.',
    description: 'Translating static text'
  },
  french: {
    title: 'Traduire du texte statique.',
    description: 'Traduire du texte statique.'
  },
  german': {
    title': 'Statischen Text übersetzen.',
    description: 'Statischen Text übersetzen.'
  }
```